### ISSUE TITLE
Lookup Tables in MAX not easy to manage

### ISSUE THREAD

**Issue by GraPhiX-Guru** [TheGameCreators/GameGuruRepo#4783]
Can we have the Lut's in MAX moved to writable area, and 'un-tied' from code.

At the moment the luts seem to be read from filternames.txt in alphabetical order and then matched to the thumbnail folder to a filter, these filters are named filter01jpg - filter999.jpg and correspond to the names filternames.txt file.

This method is not very user friendly and makes managing luts very cumbersome, it would be better if the thumbnail name matches the lut name and do away with the filternames.txt, so if I create a lut called Preset1.png the thumbnail name would be Preset1.jpg, My_First_Lut.png thumbnail would be My_First_Lut.jpg.

the default of None could be fixed but all others should be able to delete if the user wants too

This would make managing luts and Appling them a lot more user friendly especially using the application xLut

**Comment by MonkeyFrogStudio** [comment]
I totally agree. We definitely need a way to keep user LUTs separate from the stock LUTs that come with MAX. And whichever method is used, it needs to be as user-friendly as possible.

**Comment by synchromesh62** [comment]
Only recently started to take an interest in Lut's since GraPhix tool got released but yes i must admit i always like to keep anything i create for Max separate from the stock.

**Comment by davetgc** [comment]
I'd agree this would be useful, but to be fair, I've not had much time to look into it.

**Comment by Necrym59** [comment]
Now that Preben had opened up the LUT loading/changing via Lua its time this enhancement be done if possible so users can have their own writable/readable lut folders - especially with projects enhancments being done.
